### PR TITLE
update docs for TFC backend

### PR DESCRIPTION
**Pull request by megan07**
Fixes https://github.com/hashicorp/terraform-provider-google/issues/9917

If this PR is for Terraform, I acknowledge that I have:

- [x] Searched through the [issue tracker](https://github.com/hashicorp/terraform-provider-google/issues) for an open issue that this either resolves or contributes to, commented on it to claim it, and written "fixes {url}" or "part of {url}" in this PR description. If there were no relevant open issues, I opened one and commented that I would like to work on it (not necessary for very small changes).
- [x] [Generated Terraform](https://github.com/GoogleCloudPlatform/magic-modules#generating-downstream-tools), and ran `make test` and `make lint` to ensure it passes unit and linter tests.
- [x] Ensured that all new fields I added that can be set by a user appear in at least one [example](https://github.com/GoogleCloudPlatform/magic-modules/tree/master/templates/terraform/examples) (for generated resources) or [third_party test](https://github.com/GoogleCloudPlatform/magic-modules/tree/master/third_party/terraform/tests) (for handwritten resources or update tests).
- [x] [Ran](https://github.com/hashicorp/terraform-provider-google/blob/master/.github/CONTRIBUTING.md#tests) relevant acceptance tests (If the acceptance tests do not yet pass or you are unable to run them, please let your reviewer know).
- [x] Read the [Release Notes Guide](https://github.com/GoogleCloudPlatform/magic-modules/blob/master/.ci/RELEASE_NOTES_GUIDE.md) before writing my release note below.

**Release Note Template for Downstream PRs (will be copied)**

```release-note:none

```

### CODE REVIEW

File: mmv1/third_party/terraform/website/docs/guides/provider_reference.html.markdown [hunk 1]
```diff
@@ -85,6 +85,13 @@ If you are running terraform outside of Google Cloud, generate an external crede
 
 Terraform can impersonate a Google Service Account as described [here](https://cloud.google.com/iam/docs/creating-short-lived-service-account-credentials). A valid credential must be provided as mentioned in the earlier section and that identity must have the `roles/iam.serviceAccountTokenCreator` role on the service account you are impersonating.
 
+### Using Terraform Cloud as the Backend
+Place your credentials in a Terraform Cloud [environment variable](https://www.terraform.io/docs/cloud/workspaces/variables.html):
+1. Create an environment variable called `GOOGLE_CREDENTIALS` in your Terraform Cloud workspace.
+2. Remove the newline characters from your JSON key file and then paste the credentials into the environment variable value field.
+3. Mark the variable as **Sensitive** and click **Save variable**.
+All runs within the workspace will use the `GOOGLE_CREDENTIALS` variable to authenticate with Google Cloud Platform. (edited)
```
Comment (laurapacilio) [inline mmv1/third_party/terraform/website/docs/guides/provider_reference.html.markdown:93]:
```suggestion
All runs within the workspace will use the `GOOGLE_CREDENTIALS` variable to authenticate with Google Cloud Platform.
```

File: mmv1/third_party/terraform/website/docs/guides/provider_reference.html.markdown [hunk 2]
```diff
@@ -85,6 +85,13 @@ If you are running terraform outside of Google Cloud, generate an external crede
 
 Terraform can impersonate a Google Service Account as described [here](https://cloud.google.com/iam/docs/creating-short-lived-service-account-credentials). A valid credential must be provided as mentioned in the earlier section and that identity must have the `roles/iam.serviceAccountTokenCreator` role on the service account you are impersonating.
 
+### Using Terraform Cloud as the Backend
+Place your credentials in a Terraform Cloud [environment variable](https://www.terraform.io/docs/cloud/workspaces/variables.html):
+1. Create an environment variable called `GOOGLE_CREDENTIALS` in your Terraform Cloud workspace.
+2. Remove the newline characters from your JSON key file and then paste the credentials into the environment variable value field.
+3. Mark the variable as **Sensitive** and click **Save variable**.
+All runs within the workspace will use the `GOOGLE_CREDENTIALS` variable to authenticate with Google Cloud Platform.
```
Comment (laurapacilio) [inline mmv1/third_party/terraform/website/docs/guides/provider_reference.html.markdown:93]:
```suggestion
3. Mark the variable as **Sensitive** and click **Save variable**.

All runs within the workspace will use the `GOOGLE_CREDENTIALS` variable to authenticate with Google Cloud Platform.
```

File: mmv1/third_party/terraform/website/docs/guides/getting_started.html.markdown [hunk 3]
```diff
@@ -173,6 +173,13 @@ You supply the key to Terraform using the environment variable
 export GOOGLE_APPLICATION_CREDENTIALS={{path}}
 ```
 
+### Using Terraform Cloud as the Backend
+You need to use a different [environment variable](https://www.terraform.io/docs/cloud/workspaces/variables.html) name to store your credentials in Terraform Cloud.
+1. Create an environment variable called `GOOGLE_CREDENTIALS` in your Terraform Cloud workspace.
+2. Remove the newline characters from your JSON key file and then paste the credentials into the environment variable value field.
+3. Mark the variable as **Sensitive** and click **Save variable**.
+All runs within the workspace will use the `GOOGLE_CREDENTIALS` variable to authenticate with Google Cloud Platform.
```
Comment (laurapacilio) [inline mmv1/third_party/terraform/website/docs/guides/getting_started.html.markdown:181]:
```suggestion
3. Mark the variable as **Sensitive** and click **Save variable**.

All runs within the workspace will use the `GOOGLE_CREDENTIALS` variable to authenticate with Google Cloud Platform.
```

File: mmv1/third_party/terraform/website/docs/guides/getting_started.html.markdown [hunk 4]
```diff
@@ -181,6 +181,13 @@ quota or billing issues which don't seem to apply to you, you may want to set
 -> Remember to add this line to a startup file such as `bash_profile` or
 `bashrc` to store your credentials across sessions!
 
+### Using Terraform Cloud as the Backend
+You need to use a different [environment variable](https://www.terraform.io/docs/cloud/workspaces/variables.html) name to store your credentials in Terraform Cloud.
+1. Create an environment variable called `GOOGLE_CREDENTIALS` in your Terraform Cloud workspace.
+2. Remove the newline characters from your JSON key file and then paste the credentials into the environment variable value field.
+3. Mark the variable as **Sensitive** and click **Save variable**.
+All runs within the workspace will use the `GOOGLE_CREDENTIALS` variable to authenticate with Google Cloud Platform.
```
Comment (laurapacilio) [inline mmv1/third_party/terraform/website/docs/guides/getting_started.html.markdown:189]:
```suggestion
3. Mark the variable as **Sensitive** and click **Save variable**.

All runs within the workspace will use the `GOOGLE_CREDENTIALS` variable to authenticate with Google Cloud Platform.
```

**Comment by laurapacilio** [comment]
@googlebot I fixed it.

**Review by c2thorn — APPROVED** [review]

**Pull request merged**